### DISCUSSION TITLE
New Developer, Need Help Building App from Source

### DISCUSSION THREAD

**Discussion by svvvst** [florisboard/florisboard#1515] [Q&A] [answered]
Hey, I've never built an app from source code and am having difficulty getting this to run. It appears that some files are missing? e.g. icu4c folder.

**Comment by svvvst** [comment]
I was able to get this running by pasting the files from the other repo, still wondering if there is an easier way to do this.

**Comment by patrickgold** [comment, marked as answer]
You need to initialize all git submodules:

`git submodule update --init --recursive`

This way you no not need to copy any external repo files yourself.
